aspnetcore/fundamentals/minimal-apis/openapi.md

```diff
@@ -77,52 +77,143 @@ ASP.NET Core provides the [`Microsoft.AspNetCore.OpenApi`](https://www.nuget.org
 
 When using [`Swashbuckle.AspNetCore`](https://www.nuget.org/packages/Swashbuckle.AspNetCore/) with `Microsoft.AspNetCore.OpenApi`, `Swashbuckle.AspNetCore` 6.3.1 and later must be used. Version 6.4.0 or later is required if you want to [provide endpoint summary or description text](#add-endpoint-summary-or-description).
 
-### `WithOpenApi` call on endpoints adds an OpenAPI annotation
+### Add OpenAPI annotations to endpoints via `WithOpenApi`
 
-Calling [`WithOpenApi`](https://github.com/dotnet/aspnetcore/blob/8a4b4deb09c04134f22f8d39aae21d212282004f/src/OpenApi/src/OpenApiRouteHandlerBuilderExtensions.cs) on an endpoint without parameters adds an [OpenAPI annotation](https://github.com/dotnet/aspnetcore/blob/8a4b4deb09c04134f22f8d39aae21d212282004f/src/OpenApi/src/OpenApiRouteHandlerBuilderExtensions.cs#L65-L82) to the endpoints metadata. This metadata can be:
+Calling [WithOpenApi](/dotnet/api/microsoft.aspnetcore.builder.openapiendpointconventionbuilderextensions.withopenapi) on the endpoint to the endpoints metadata. This metadata can be:
 
 * Consumed in third-party packages like [Swashbuckle.AspNetCore](https://www.nuget.org/packages/Swashbuckle.AspNetCore/).
 * Displayed in the Swagger user interface or in YAML or JSON generated to define the file.
 
 [!code-csharp[](7.0-samples/todo/Program.cs?name=snippet_withopenapi&highlight=9)]
 
-#### Call `WithOpenApi` with parameters
+#### Modify the OpenAPI annotation in `WithOpenApi`
 
-The [`WithOpenApi`](https://github.com/dotnet/aspnetcore/blob/8a4b4deb09c04134f22f8d39aae21d212282004f/src/OpenApi/src/OpenApiRouteHandlerBuilderExtensions.cs#L49) method accepts a function that can be used to modify the OpenAPI annotation. For example, in the following code, a description is added to the first parameter of the endpoint:
+The [WithOpenApi](/dotnet/api/microsoft.aspnetcore.builder.openapiendpointconventionbuilderextensions.withopenapi) method accepts a function that can be used to directly modify the OpenAPI annotation. For example, in the following code, a description is added to the first parameter of the endpoint:
 
 [!code-csharp[](7.0-samples/todo/Program.cs?name=snippet_withopenapi2&highlight=9-99)]
 
+### Add operation IDs to Open API
+
+Operation IDs are used to uniquely identify a given endpoint in OpenAPI. The [`WithName`](/dotnet/api/microsoft.aspnetcore.builder.routingendpointconventionbuilderextensions.withname) extension method can be used to set the operation ID used for a method.
```

Comment:
Do you think there's value in linking to the spec docs? e.g.

```suggestion
Operation IDs are used to uniquely identify a given endpoint in OpenAPI. The [`WithName`](/dotnet/api/microsoft.aspnetcore.builder.routingendpointconventionbuilderextensions.withname) extension method can be used to set the [operationId](https://spec.openapis.org/oas/latest.html#fixed-fields-7) used for a method.
```

Reply:
Linking to the spec is really hard. Ideally, I think linking to the `Microsoft.OpenApi` docs would provide the most tangible value for users ("oh this is the type I need to pass in"). I filed an issue to that effect over at https://github.com/microsoft/OpenAPI.NET/issues/1026.

Reply:
Fair enough. We should make a call on how to refer to the fields then, i.e. verbatim like "operationId", or in descriptive fashion like "operation ID". Personally I prefer the former as it matches what is being set and how it appears in the produced document, appears in spec docs, etc.

Reply:
TBH, I prefer the later because there is a disparity between how the operation ID is set in the OpenAPI.NET models (**O**perationId) versus how it appears in the serialized document and the UI (**o**operationId) so I think the descriptive reference is the most implementation neutral. Also the code sample already shows how to set it.